rpcs3/rpcs3qt/log_frame.h

```diff
@@ -56,7 +56,7 @@ private Q_SLOTS:
 	bool m_stack_log{};
 	bool m_stack_tty{};
 
-	fs::file m_tty_file;
+	u64 m_tty_time = 0;
```

Comment:
if you're already removing the file member, why do you keep the File include in the header ?

Reply:
For compilation, other cpp files using this header assuming ng File.h is included. Can be resolved in some include cleanup pr.

Reply:
Fix it in this PR. A follow up PR is really. Or necessary for this trivial task